### DISCUSSION TITLE
Mageworx Shipping Calculator Plugin integration with breeze blank (breeze experience enabled)

### DISCUSSION THREAD

**Discussion by shams7777** [breezefront/community#147] [3. Integrations]
https://marketplace.magento.com/mageworx-module-shippingcalculator.html
i want to integrate this plugin with breeze blank theme or if there is another plugin on swissuplabs hich can calculate shipping costs.

**Comment by dimdimich** [comment]
Thank you for using Breeze. Could you possibly do the Breeze community a favor and submit a ticket with Mageworx to request compatibility from their end first?  The more people that ask Mageworx, the more likely it is that support will be implemented by the extension vendor.

**Reply by shams7777** [reply]
I've already contacted the support team!
is there any plugin swissuplabs which can replace mageworx shipping calculator and  do shipping calculation ?
